app.js

```diff
@@ -1,123 +1,236 @@
 'use strict';
 
 var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '8pm', 'total'];
-//15 hours in day
 
-// create ul tag
-// create one li tag
-// fill li tag innertext w/ information
-// append li tag to ul as child
-// append ul to body
-// var element = document.createElement(tagName[, options]);
+var body = document.getElementsByTagName('body')[0];
 
-
-var firstPike = {
+var firstAndPike = {
   name: '1st and Pike',
   minCust: 23,
   maxCust: 65,
-  avgCookie: 6.3,
-  hourlysales: [],
-
-  randomCustomerPerHour: function(){
-    return Math.floor(Math.random() * ((this.maxCust - this.minCust) + this.minCust));
+  avgCookies: 6.3,
+  salesArr: [],
+  randCust: function(){
+    //generate random # of cust
+    return Math.floor(Math.random() * (this.maxCust - this.minCust + 1) + this.minCust);
+    console.log(this.randCust);
+  },
+  cookiesSold: function(){
+    var total = 0;
+    for(var i = 0; i < storeHours.length -1; i++) {
+    //-1 of storeHours array [total]
```

Comment:
Make this line more descriptive so that someone who doesn't already know the codebase will know what this comment means.